src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -420,51 +449,95 @@ object DeployPlugin extends AutoPlugin {
     overrides.withFallback(rcConfig.withFallback(targetConfig)).resolve
   }
 
-  /** Translates the given deploy Config object into a flat Map[String, String]
-    * containing all the relevant deployment keys.
-    *
-    * Exits the program if any of the keys required for deployment are missing.
-    *
-    * @param targetName the name of the target to print on error
-    * @param targetConfig the config to parse
+  /** Check that all environment variables required for deployment are set,
+    * and exit the program if any are missing.
     */
-  def validateAsMap(targetName: String, targetConfig: Config): Map[String, String] = {
-    // Validate that the target has all the required environment variables.
+  def validateEnv(): Unit = {
     val requiredEnv = Seq("AWS_PEM_FILE")
-    for (variable <- requiredEnv) {
-      val value = sys.env.get(variable).getOrElse {
-        throw new IllegalArgumentException(s"Environment variable ${variable} undefined.")
-      }
+    requiredEnv foreach { variable =>
```

Comment:
I'd leave this in for now, unless you want to verify the daily deploy without it. I think the evaluation framework may still rely on it, since it's using our old AMI.